src/controller/user.py

```
@@ -315,3 +364,362 @@ async def reactivate_oldboy_ctrl(session: SessionDep, user: User) -> None:
+            query = query.where(User.name == name)
+        if phone:
+            query = query.where(User.phone == phone)
+        if student_id:
+            query = query.where(User.student_id == student_id)
+        if user_role:
+            query = query.where(User.role == get_user_role_level(user_role))
+        if status:
+            query = query.where(User.status == status)
+        if discord_id is not None:
+            if discord_id == "":
+                query = query.where(User.discord_id == None)
```

Ruff (0.14.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)